e2e/pages/Onboarding5.po.js

```diff
@@ -2,9 +2,9 @@
 const buttonLabel = 'Next';
 const screenTitleTestId = 'Header';
 const screenSubtitle = 'Subheader';
-const screenshotText = 'Onboarding - Page 5';
+const screenshotText = 'Onboarding - Page 6';
 
-class Onboarding5 {
+class Onboarding6 {
```

Comment:
I'm afraid of this 4->5 5->6 stuff.. is it necessary?

Reply:
nope - should we just remove for the screenshot texts?